src/lr_reduction/output.py

```
@@ -200,7 +202,7 @@ def read_file(file_path):
     """
     _meta = dict()
     with open(file_path, "r") as fd:
-        for l in fd.readlines():
+        for l in fd:
```

Ruff (0.14.10)

Ambiguous variable name: `l`

(E741)